### DISCUSSION TITLE
Night timer for bluetooth

### DISCUSSION THREAD

**Discussion by Darkmagister** [UnicornsOnLSD/finamp#918] [General]
I like the night timer that stop the music after some times, but I was thinking if it was possible to have something that will also disconnect from a specific bt device.

Here my situation I have some Alexa but I don't like their music service so I will connect my phone to it and listen to some music to go to sleep with a timer, but in the morning I don't like that the phone is still connected to it and the alarm play on it, there is something o it would be possible to make it disconnect from a specific bt device when the timer passed? (Not turning off the bt if possible as I also have a smartwatch and I would still like to have it connected).

What is your thoughts?

**Comment by Chaphasilor** [comment]
I'm honestly not sure if it's possible. I recently checked some bluetooth-related stuff with Flutter and it was rather disappointing. Seems like on iOS it's not possible at all, and on Android we need to write our own Android code to have a chance.  

I actually have the same issue as you and just opted to have my smart home turn of the amplifier if an alarm is about to go off and my phone is connected to it.  

You could try some other automation tools (MacroDroid/Tasker on Android or Siri Shortcuts on iOS) to achieve something similar.

If anyone can figure out a way to do this from Finamp instead, I'd be happy to implement it!
